Prepare repository to be de-coupled from aspnet/AspNetCore (#416)
aspnet/Common: README.md

Security issues and bugs should be reported privately, via email, to the Microsoft Security Response Center (MSRC)  secure@microsoft.com. You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Further information, including the MSRC PGP key, can be found in the [Security TechCenter](https://technet.microsoft.com/en-us/security/ff852094.aspx).

## Related projects

These are some other repos for related projects:

* [.NET Core](https://github.com/dotnet/core) - a cross-platform, open-source .NET platform
* [ASP.NET Core](https://github.com/aspnet/AspNetCore) - a .NET Core framework for building web apps
* [Entity Framework Core](https://github.com/aspnet/EntityFrameworkCore) - data access technology

## Code of conduct

This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).  For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

## Community forks

> Some parts of this project are no longer under active development, but are maintained as community forks.

Reviewer: Not sure I'd say `no longer under active development`, but maybe instead:

```suggestion
Some parts of this project have been forked by the community to add additional functionality:
```
